x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/credential_access_cmdline_dump_tool.json

```diff
@@ -91,5 +91,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 8
+  "version": 101
```

Comment:
@terrancedejesus shouldn't this just be 100? I think our python logic is working, but since this is our first time bumping to `100`, shouldn't they all just go to all. (even if we've made updates in between)

Reply:
nope, this is correct as it represents a change in the rule. Since we have not locked, this version will not bump for every subsequent change until after locked. This is as designed